scripts/ci/gke.sh

```diff
@@ -208,7 +209,7 @@ create_cluster() {
             fi
             info "Timed out"
             info "Attempting to delete the cluster before trying another zone"
-            gcloud container clusters delete "${CLUSTER_NAME}" || {
+            retry 10 true gcloud container clusters delete "${CLUSTER_NAME}" --async || {
```

Comment:
I don't think this will help. In the zone failure the cluster create appears stuck. (Already waited `timeout 830` and then sleep 60 * 20. When the delete fails the next zone worked which is the desired outcome.